scripts/SANS/isis_reduction_steps.py

```diff
@@ -4282,7 +4282,7 @@ def execute(self, reducer, workspace):
             Reads the geometry information stored in the workspace
             but doesn't replace values that have been previously set
         """
-        _reducer = reducer  # noqa
+        _ = reducer  # noqa
```

Comment:
Ah I see. Sorry, I had missed the function arguments. Could you just have the signature as `def execute(self, _, workspace)` to indicate it is not used?